src/Datadog.Trace.ClrProfiler.Managed/ScopeFactory.cs

```diff
@@ -19,6 +19,22 @@ internal static class ScopeFactory
 
         private static readonly Vendors.Serilog.ILogger Log = DatadogLogging.GetLogger(typeof(ScopeFactory));
 
+        public static Scope GetActiveHttpScope(Tracer tracer)
+        {
+            var scope = tracer.ActiveScope;
+
+            var parent = scope?.Span;
+
+            if (parent != null &&
+                parent.Type == SpanTypes.Http &&
+                parent.GetTag(Tags.InstrumentationName) != null)
```

Comment:
What is the point of the InstrumentationName check?

Reply:
This check already exists in `CreateOutboundHttpScope`. First, to make sure we only create one automatic instrumentation span when the framework has nested calls like WebRequest -> HttpMessageHandler in .NET Core, we check the span type. However, we had a case where users created manual spans themselves, and this caused us to bail out of automatic instrumentation. We don't want to bail out in this scenario because manual instrumentation may not have propagated the context, so we'll only bail out when the parent span is an automatic instrumentation span.

Reply:
I guess this explanation is worthy of being documented in the code 😅